src/screens/ProfileScreen.js

```diff
@@ -1,15 +1,225 @@
-import React from 'react'
-import { StyleSheet, Text, View } from 'react-native'
+import React, { useState } from 'react'
+import { SafeAreaView, ScrollView, StyleSheet, View } from 'react-native'
 import i18n from 'i18next'
+import { LinearProgress, ListItem, Icon } from 'react-native-elements'
+import { useTranslation } from 'react-i18next'
+import Colors from '../constants/Colors'
+import { TTSengine } from '../components/Tts'
+import * as data from '../profileData.json'
+
```

Comment:
iterate `data` and replace color (success, info etc.) with color value from `Colors`